### PR TITLE
Use native checkbox input for auto roll checkboxes.

### PR DESCRIPTION
**Pull request by schultzcole**
This is far more likely to be compatible with other sheet modules, and will not display weird behavior like submitting the page.

mre-auto-roll-checkboxes

### CODE REVIEW

File: scripts/hooks/render-item-5e-sheet.mjs [hunk 1]
```diff
@@ -4,7 +4,10 @@ import { FormulaGroupConfig } from "../apps/formula-group-config.mjs";
 Hooks.on("renderItemSheet5e", (itemSheet, html, _) => {
     // Add formula group config button
     const tooltip = game.i18n.localize(`${MODULE_NAME}.FORMULA-GROUP.DialogTitle`);
-    html.find(".tab.details .damage-header").prepend(`<a title="${tooltip}" class="config-formula-groups"><i class="fas fa-tasks"></i></a>`);
+    const damageHeader = html.find(".tab.details .damage-header");
+    damageHeader.wrap(`<div class="mre-damage-header-container">`);
```
Comment (akrigline) [inline scripts/hooks/render-item-5e-sheet.mjs:8]: Possible this won't exist, e.g. for Tool items. Should this be option chained or is jquery smart enough to handle that?

Comment (schultzcole) [reply scripts/hooks/render-item-5e-sheet.mjs:8]: I think jQuery will handle that, but I will double check tomorrow.

Comment (schultzcole) [reply scripts/hooks/render-item-5e-sheet.mjs:10]: Yes, jQuery will just do nothing if those elements are not present.

**Comment by schultzcole** [comment]
I did some work to further handle cases where the item sheet is not editable.

**Review by akrigline — APPROVED** [review]
awesome, nice simplification to use `indeterminate`

**Pull request merged**